src/common/AndroidUtils.ts

```diff
@@ -56,13 +56,6 @@ export class AndroidSDKUtils {
         return !AndroidSDKUtils.packageCache.isEmpty();
     }
 
-    public static executeCommand(
```

Comment:
There's no point in having this function here when all it does is call into another function 🙂